test/integration/scripts/create_pkcs_store.sh

```diff
@@ -97,6 +97,15 @@ echo "importing rsa2048 key under token 'import-keys'"
 openssl genrsa -out private.pem 2048
 tpm2_ptool.py import --privkey='private.pem' --algorithm=rsa --key-label="imported_key" --label="import-keys" --userpin=anotheruserpin --path=$TPM2_PKCS11_STORE
 
+#Passwordless Primary
+tpm2_ptool.py init --path=$TPM2_PKCS11_STORE
+##Add token
+tpm2_ptool.py addtoken --pid=3 --sopin=mysopin --userpin=myuserpin --label=pwless --path $TPM2_PKCS11_STORE
+###Add aes key under token
+tpm2_ptool.py addkey --algorithm=aes256 --label=pwless --userpin=myuserpin --path=$TPM2_PKCS11_STORE
+##Verify setup
+tpm2_ptool.py verify --sopin=mysopin --userpin=myuserpin --label=pwless --path=$TPM2_PKCS11_STORE
```

Comment:
A bunch of tests are failing, probably hard-code the number of tokens expected. Please ensure make check passes.

Reply:
Thanks for the pointer